config.example.py

```diff
@@ -31,7 +33,9 @@
       'db_review': 'db/*',
       'css_review': 'app/assets/stylesheets/global/*',
       'js_review': 'node_modules/*',
-      'ops_review': ['config/nginx/*', 'config/settings/*']
+      'ops_review': ['config/nginx/*', 'config/settings/*'],
+      # Only matches files in this folder, not any subfolder
+      'test_review': re.compile(r'test\/[a-z_]+.rb'),
```

Comment:
Since we're using match and not findall, there is no need to specify the start and end of a string.

<img width="844" alt="screen shot 2017-01-09 at 10 14 32 am" src="https://cloud.githubusercontent.com/assets/1444314/21761686/e5f53e8c-d654-11e6-91d0-a2f7b0a2abd9.png">